mathics/core/expression.py

```diff
@@ -560,8 +603,8 @@ def elements(self):
         return self._elements
 
     @elements.setter
-    def elements(self, values):
-        self._elements = self._build_elements(values)
+    def elements(self, values: Iterable, **kwargs: Optional[dict]):
```

Comment:
I guess that the setter never is called with this optional value.

Reply:
Let me state a principle before coming back to whether to add the `kwargs` argument. 

In general, the kinds of things that need to get done takes precedence over mechanisms and syntactic sugar.  The whole idea of allowing elements to be set might be wrong, and might be avoided with a better interface and possibly better modularity of the code. 

The reason `kwargs` was added was to allow being able to set elements and specify properties without going through the Expression  constructor.   The place that used that could have done so probably the same and skipped a call by calling `self._build_elements()` directly and then probably the leading `_` in `_build_elements()` should be removed. 

So  `**kwargs` could be removed which would make the elements setter probably more normal and less surprising. 

As with the other comments here, all of this is futzing around at the expense of devoting time to more important places.

It's not that I don't think this shouldn't be addressed, I can make note of this and after a pass is made over evaluation if such things are marked or there are alternative interfaces, then we should come back to this and address these kinds of things.

Reply:
Removed in a1280cc9cec1c0335630ca1e5d5e7465ab8a7095